src/App.js

```diff
@@ -0,0 +1,109 @@
+import { Component } from 'react';
+import AppHeader from './AppHeader';
+import ItemsList from './ItemsList';
+import AppFooter from './AppFooter';
+
+class App extends Component {
+    constructor() {
+        super()
+        this.state = {
+            inputfield: '',
+            inputs: [],
+            saveButtonClassActive: false
+        };
+    }
+
+    componentDidMount() {
+        const storageItems = localStorage.getItem('items');
```

Comment:
can you please put 'items' in some `const` and use it here and in the place where you save it?

Reply:
I created in class constructor `this.localStorageItems` to store the 'items'.